src/ogs.styl

```diff
@@ -139,6 +139,11 @@ light.weak-loss                         = lighten(light.loss, 25%)
 light.strong-win                        = darken(light.win, 25%)
 light.strong-loss                       = darken(light.loss, 25%)
 
+light.strong-w-text                     = #E4DCDC
+light.strong-l-text                     = #E4DCDC
```

Comment:
#fff is to bright? (#fff is the background color of the light theme. It would be consistent with my suggestions for dark theme below)

Reply:
Color has been changed to #fff